test/e2e/performanceprofile/functests/10_performance_ppc/10_ppc_suite_test.go

```diff
@@ -0,0 +1,39 @@
+package __performance_ppc
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	. "github.com/onsi/gomega"
+	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+)
+
+const (
+	DefaultPodmanBinaryPath = "/usr/bin/podman"
+)
+
+var _ = BeforeSuite(func() {
+	var err error
+	By("Check podman binary exists")
+	path, err := exec.LookPath(DefaultPodmanBinaryPath)
+	if err != nil {
+		Skip(fmt.Sprintf("%s doesn't exists", DefaultPodmanBinaryPath))
+	}
+	testlog.Infof("Podman binary executed from path %s", path)
+	By("Checking Environment Variables")
+	testlog.Infof("NTO Image used: %s", testutils.NtoImageRegistry)
+	if testutils.MustGatherDir == "" {
+		Skip("set env variable MUSTGATHER_DIR to ocp mustgather directory")
```

Comment:
hmm, i don't want to use default path because we run the tests on different environment, disconnected cluster, local laptop, clusterbot, downstream ci and each ci runs mustgather in a different fashion , where the mustgather is extracted in different location .

If we use an environment variable, we can modify the CI to use environment variable pointing to the extracted mustgather directory.

Reply:
ok makes sense, thanks for clarifying.